charts/vllm/values.yaml

```diff
@@ -2,6 +2,16 @@ replicaCount: 1
 # Change this if you want to serve another model
 model: mistralai/Mistral-7B-Instruct-v0.1
 
+readManyPVC:
```

Comment:
Maybe add a comment about the known-cloud support for this... I think it is only GKE right now.

Reply:
fixed, added it's GKE only for now